keps/349-all-or-nothing/README.md

```diff
@@ -223,7 +238,7 @@ type WaitForPodsReady struct {
 
 We introduce a new workload condition, called `PodsReady`, to indicate
 if the workload's startup requirements are satisfied. More precisely, we add
-the condition when `job.status.ready + job.status.succeeded` is greater or equal
+the condition when `job.status.ready + job.status.uncountedTerimnatedPods + job.status.succeeded` is greater or equal
```

Comment:
```suggestion
the condition when `job.status.ready + job.status.uncountedTerminatedPods + job.status.succeeded` is greater or equal
```

Reply:
I would also suggest to add a note why we do this somewhere in the KEP update. It could be Design Details or Notes section.

Reply:
Could you elaborate on that? It is in the Design Details section

Reply:
I see, the formula is in Design Details, but I would like to have a short note **why** we have `job.status.uncountedTerimnatedPods` here. Maybe something along the lines:
Note that we count `job.status.uncountedTerimnatedPods` - this is meant to prevent flickering of the `PodsReady` condition when pods are transitioning to the Succeeded state.

Reply:
Done